core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java

```diff
@@ -482,22 +482,6 @@ public static class Upgrade11to12 {
     }
   }
 
-  /**
-   * Contains additional metadata in a reserved area not for tablets
-   */
-  public static class ReservedSection {
-    private static final Section section = new Section(RESERVED_PREFIX, true, null, false);
-
-    public static Range getRange() {
-      return section.getRange();
-    }
-
-    public static String getRowPrefix() {
-      return section.getRowPrefix();
-    }
-
-  }
-
```

Comment:
This class serves as schema documentation. We don't have a need to scan the whole reserved range, so we don't use it directly, but it should still exist to provide a mechanism in code to evaluate the schema of the metadata table.